commons-extension-webapp/src/main/webapp/login/jsp/login.jsp

```diff
@@ -135,16 +135,17 @@ PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
               if (errorData != null) {
 						if (org.exoplatform.web.login.LoginError.DISABLED_USER_ERROR == errorData.getCode()) {
         	 %>
-          <div class="signinFail"><i class="uiIconError"></i><%=res.getString("UILoginForm.label.DisabledUserSignin")%></div>
+          <div class="signinFail" role="alert"><i class="uiIconError"></i><%=res.getString("UILoginForm.label.DisabledUserSignin")
```

Comment:
I think that this should be on one line

Reply:
I just made the backport @rdenarie what should I change?

Reply:
hum ok, so let as is